docs/compiler.md

```diff
@@ -33,6 +33,13 @@ a dialect of Go rather than a complete port of the language:
    it's up to the programmer whether assert can be performed successfully.
```

Comment:
Format of the commit message is questionable. @AnnaShaleva would not like that.

Reply:
Do you think it violates https://github.com/nspcc-dev/.github/blob/master/git.md#commit-messages?

Reply:
yes,

> The following format is expected:
> 
> subsystem: short description (up to 80 symbols, better up to 60)
> 
> Long description with references, links...

in the first line there are no links or refs. At least they are not mentioned in https://github.com/nspcc-dev/.github/blob/master/git.md#commit-messages

Reply:
While I think it's compliant (there are no limits of what short description can or can not include as long as it's short) and there are numerous commits like f4b61b5a7, b1bb12df4, d2a716221, eade327b9 and others, I can at the same time say that @AnnaShaleva has _never_ wrote a short description like that in ~1715 commits that I can see in this repo (~116 commits in neo-bench are the same). I think it's still OK, but maybe we can have our policies improved one way or another.

Reply:
> in the first line there are no links or refs. 

I'm OK with the commit formatting presented in this PR, links are allowed both in the commit header and in the body.